source/MRViewer/MRRibbonMenu.h

```diff
@@ -141,8 +141,27 @@ class MRVIEWER_CLASS RibbonMenu : public ImGuiMenu
 
     MRVIEWER_API virtual void drawCustomObjectPrefixInScene_( const Object& obj ) override;
 
+    enum class ShortcutCategory
+    {
+        Info,
+        Edit,
+        View,
+        Scene,
+        Objects,        
+        Selection,
+        Count
+    };
+
+    std::map<ShortcutManager::ShortcutKey, ShortcutCategory> _shortcutsByCategory;
```

Comment:
better move category logic to shortcut manager (add category field in `ShortcutCommand` struct)